### ISSUE TITLE
JAP - The 'Development Programs' decisions are still existing despite the Economic Crisis has ended

### ISSUE THREAD

**Issue by Ikidikid** [TNODevs/TNO-Bug-Tracker#3779]
### Operating System

Windows 10 Enterprise

### HOI4 version

1.15.2

### TNO version

1.6.0a

### Do you have other mods in use?

None

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Files

[JAP.zip](https://github.com/user-attachments/files/18048274/JAP.zip)

### Describe the bug

When the Yasuda Crisis ended, the 'Development Programs' decisions are still exist and these decisions can continue to appear again after the decision finished. Which is means that Japan can gain more economic buffs or other effect buffs in the 1970s. 

### Steps to reproduce / Cause of the issue

1. The Yasuda Crisis triggers.
2. Has the 'Development Programs' decisions.
3. Finished national focus tree.
4. The Yasuda Crisis ended.
5. Has the Ending screen.

### Expected behaviour

Probably, oversight.

### Screenshots

![Image_1](https://github.com/user-attachments/assets/89da6c77-b9b3-49d5-86df-ae663e565dbc)

**Comment by Marok132** [comment]
@Jack121Q I'm sorry for a ping but it works as intended, doesn't it?

**Comment by Jack121Q** [comment]
Yes, these are meant to last the entire game. That should be clear from reading the description of the decision category.
